data/building-hours/1-5-sayles-equivalency.yaml

```
@@ -5,21 +5,19 @@ category: Food
 schedule:
   - title: Breakfast
     hours:
-      - {days: [Mo, Tu, We, Th, Fr], from: '8:00am', to: '10:30am'}
-      - {days: [Sa], from: '9:00am', to: '10:30am'}
+      - {days: [Mo, Tu, We, Th, Fr], from: '7:30am', to: '10:00am'}
+      - {days: [Sa], from: '9:00am', to: '10:00am'}      
 
   - title: Lunch
     hours:
-      - {days: [Mo, Tu, We, Th, Fr], from: '11:30am', to: '5:00pm'}
-      - {days: [Sa], from: '10:30am', to: '5:00pm'}
-      - {days: [Su], from: '12:00pm', to: '5:00pm'}
+      - {days: [Mo, Tu, We, Th, Fr], from: '11:00am', to: '6:00pm'}
+      - {days: [Sa], from: '11:00am', to: '6:00pm'}
+      - {days: [Su], from: '12:00pm', to: '6:00pm'}
 
-  - title: Dinner
+  - title: Late Night
     hours:
-      - {days: [Mo, Tu, We, Th], from: '5:00pm', to: '12:00am'}
-      - {days: [Fr], from: '5:00pm', to: '1:00am'}
-      - {days: [Sa], from: '5:00pm', to: '1:00am'}
-      - {days: [Su], from: '5:00pm', to: '12:00am'}
+      - {days: [Mo, Tu, We, Th, Fr], from: '6:00pm', to: '10:00pm'}
+      - {days: [Sa, Su], from: '6:00pm', to: '10:00pm'}
```

```suggestion
      - {days: [Mo, Tu, We, Th, Fr, Sa, Su], from: '6:00pm', to: '10:00pm'}
```

It looks like all days are the same hours - is that correct?